miasm2/core/graph.py

```diff
@@ -149,3 +159,44 @@ def dot(self):
                                                 self.edge2str(a, b))
         out += "}"
         return out
+
+
+
+    def compute_dominators(self):
+        """Compute dominators of the graph"""
+        nodes = self.nodes()
+        heads = self.heads()
```

Comment:
Is this a comment on code optimisation?
Do you think it is more interesting to use at least 4 times an iterator generation than generating an object once and referencing this non modified object?
The iterator will be called at least 4 times:
* `heads = self.heads()`
* `for n in heads:`
* `todo = nodes.difference(heads)`
* `if node in heads:` (in a loop)